pom.xml

```diff
@@ -68,10 +66,9 @@
             <plugin>
                 <artifactId>maven-release-plugin</artifactId>
                 <configuration>
-                    <tagNameFormat>@{project.version}</tagNameFormat>
```

Comment:
Since one GitHub repository will be hosting multiple releases for multiple artifacts, we need to re-add the artifactId to the version tag.  For example, releasing the parent POM to version 0.0.2 will create a tag named "tds-common-parent-0.0.2"